src/main/java/se/sundsvall/supportmanagement/api/ErrandsResource.java

```diff
@@ -73,10 +74,11 @@ class ErrandsResource {
 	ResponseEntity<Void> createErrand(
 		@Parameter(name = "namespace", description = "Namespace", example = "MY_NAMESPACE") @Pattern(regexp = NAMESPACE_REGEXP, message = NAMESPACE_VALIDATION_MESSAGE, groups = OnCreate.class) @PathVariable final String namespace,
 		@Parameter(name = "municipalityId", description = "Municipality id", example = "2281") @ValidMunicipalityId(groups = OnCreate.class) @PathVariable final String municipalityId,
+		@Parameter(name = "referred_from", description = "Reference of origin", example = "portal") @RequestParam(name = "referred_from", required = false) final String referredFrom,
```

Comment:
Make it more clear that referred_from should be a reference to an errand?

`@Parameter(name = "referred_from", description = "Reference of origin", example = "portal")`
->
`@Parameter(name = "referred_from", description = "Referenced from errand", example = "ABC-123456")`